main.js

```diff
@@ -0,0 +1,95 @@
+
+let w = 40;
```

Comment:
Think it'd help to put this part in a function, like "createBoard" or something.